kclvm/tools/src/LSP/src/hover.rs

```diff
@@ -36,16 +36,28 @@ pub(crate) fn hover(
                         // attr2? type
                         // ```
                         let schema_ty = ty.into_schema_type();
-                        docs.push(schema_ty.schema_ty_signature_str());
+                        let schema_text = schema_ty.schema_ty_signature_str();
+
+                        for (i, ele) in schema_text.split("\n\n").enumerate() {
```

Comment:
don't split here. you can define a new function to get signature without pkg

Reply:
I have made a new function that returns signature without pkg in mod.rs